webapp/packages/plugin-connections-administration/src/Administration/Connections/ConnectionsTable/ConnectionsTable.tsx

```diff
@@ -42,6 +43,15 @@ export const ConnectionsTable = observer<Props>(function ConnectionsTable({
   expandedItems,
 }) {
   const translate = useTranslate();
+  const projectService = useService(ProjectsService);
+  const projectsLoader = useResource(ConnectionsTable, ProjectInfoResource, CachedMapAllKey);
+  const displayProjects = getComputed(() => projectService
+    .activeProjects
+    .filter(project => isGlobalProject(project) || isSharedProject(project)).length > 1);
+
+  function getProjectName(projectId: string) {
```

Comment:
You could make it a variable inside the loop
const projectName = displayProjects ? projectsLoader.resource.get(projectId)?.name : undefined;